integrations/repo_test.go

```diff
@@ -73,13 +73,13 @@ func testViewRepo(t *testing.T) {
 			fileName:   "doc",
 			commitID:   "2a47ca4b614a9f5a43abbd5ad851a54a616ffee6",
 			commitMsg:  "init project",
-			commitTime: time.Date(2017, time.June, 14, 13, 54, 21, 0, time.UTC).Format(time.RFC1123),
+			commitTime: time.Date(2017, time.June, 14, 13, 54, 21, 0, time.Now().Location()).Format(time.RFC1123),
```

Comment:
time.Local ?

Reply:
time.Local is a good hint ... but it failed localy:
```
-  commitTime: (string) (len=30) "Wed, 14 Jun 2017 13:54:21 CEST"
+  commitTime: (string) (len=30) "Wed, 14 Jun 2017 15:54:21 CEST"
```
so I have to change more ... (I'll combine my first (not published version and your hint) :)

Reply:
tested localy again this should work :)